assets/app/scripts/services/data.js

```diff
@@ -1034,6 +1152,9 @@ angular.module('openshiftConsole')
   DataService.prototype.url = function(options) {
     if (options && options.resource) {
       var opts = angular.copy(options);
+      if(!!options.isWebsocket) {
```

Comment:
I was just about to comment on this, running through the tests this morning and they failed:
```
Expected 
'ws://localhost:8443/api/v1/namespaces/foo/pods?watch=true' to equal 'ws://localhost:8443/api/v1/watch/namespaces/foo/pods'

Expected 
'ws://localhost:8443/api/v1/namespaces/foo/pods?resourceVersion=5&watch=true' to equal 'ws://localhost:8443/api/v1/watch/namespaces/foo/pods?resourceVersion=5'
```
though when i experiment in the browser both formats will actually generate a working WebSocket.  I'll update.